src/sentry/identity/vsts/provider.py

```diff
@@ -16,7 +17,7 @@ class VSTSIdentityProvider(OAuth2Provider):
 
     oauth_access_token_url = 'https://app.vssps.visualstudio.com/oauth2/token'
     oauth_authorize_url = 'https://app.vssps.visualstudio.com/oauth2/authorize'
-
+    oauth_redirect_url = '/extensions/vsts/setup/'
```

Comment:
The identity provider has redirect URLs abstracted from it, I think this is breaking that abstraction. For example you can see her how it works

https://github.com/getsentry/sentry/blob/ea1b6476fb261f879b962b178a95f7739dc61ffb/src/sentry/identity/oauth2.py#L144

However in this case we have access to the pipeline from the dispatched method.

Reading further though I'd like to understand more why we actually need this, checkout some of my following comments.